src/Uno.Toolkit.UI/Controls/TabBar/TabBarItem.Properties.cs

```diff
@@ -51,6 +51,17 @@ public Visibility BadgeVisibility
 			DependencyProperty.Register("BadgeValue", typeof(string), typeof(TabBarItem), new PropertyMetadata(default(string?), OnPropertyChanged));
 		#endregion
 
+		#region InfoBadge
```

Comment:
Are we planning on removing the BadgeValue and BadgeVisibility properties and introduce this as a breaking change? I feel like this might be confusing to have InfoBadge alongside the existing Badge related properties and have them work independently. 

I'd say we either get rid of them or forward their functionality to the InfoBadge property

Reply:
fyi @MartinZikmund / @agneszitte

Reply:
My vote is to remove them and document a breaking change

Reply:
I think the same @kazo0 on my side and in addition to documenting the breaking change, this commit should be properly updated to mention a breaking change as well with conventional commits.
Do you think the same  @MartinZikmund, @Xiaoy312 ?

Reply:
Technically they can be data-bound but usually not. It suffers from [this issue](https://github.com/unoplatform/uno.toolkit.ui/issues/1129) if you were to try to use the TabBarItem from a DataTemplate

Reply:
Decided to not make a breaking change with the `InfoBadge` change at the moment according to conversation we had on standup. Now, users can use both and we can deprecate `BadgeVisibility` and `BadgeValue` in later releases.